README.md

```diff
@@ -187,6 +187,29 @@ curl -X 'POST'
  | jq -C . | less -R
 ```
 
+
+#### Chunking Elements
+
+Set the `chunking_strategy` to chunk text into larger or smaller elements. Defaults to `None` with optional arg of `by_title`.
```

Comment:
Can you clarify in here that these params are defaults and not exposed yet?

Reply:
Sorry, but what do you mean by "not exposed" yet?

Reply:
Ohh... do you mean for `multipage_sections` and the other additional params?

Reply:
Yeah, perhaps a note that they'll be in the api soon, and here are the defaults for now.

Reply:
Should I go ahead and add them for this PR?

Reply:
Your call! Might as well if they're all similar.